tpl/template_embedded.go

```diff
@@ -233,7 +233,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
 {{ end }}{{ end }}
 
 <!-- Output all taxonomies as schema.org keywords -->
-<meta itemprop="keywords" content="{{ range $plural, $terms := .Site.Taxonomies }}{{ range $term, $val := $terms }}{{ printf "%s," $term }}{{ end }}{{ end }}" />
+<meta itemprop="keywords" content="{{ if .IsPage}}{{ range $index, $tag := .Params.tags }}{{ $tag }},{{ end }}{{ else }}{{ range $plural, $terms := .Site.Taxonomies }}{{ range $term, $val := $terms }}{{ printf "%s," $term }}{{ end }}{{ end }}{{ end }}" />
```

Comment:
Spotted another issue. This code is outputting hyphenated keyphrases, and the output should be humanized (preferably in lower case), e.g.

```
{{ lower (humanize $term) }}
```